lib/install/operation.go

```diff
@@ -232,19 +227,6 @@ func (i *Installer) getClusterStatus() (*status.Status, error) {
 	return status, nil
 }
 
-// upsertAdminAgent creates an admin agent for the cluster being installed
-func (i *Installer) upsertAdminAgent(clusterName string) error {
```

Comment:
Moved to lib/ops/opsservice to the point of creating the cluster where it's more context-specific and resume-friendly.